### ISSUE TITLE
Integrating Persephone into a production ML pipeline

### ISSUE THREAD

**Issue by toonday** [persephone-tools/persephone#235]
I am trying to integrate models built with Persephone into my ML pipeline. I started by developing models using the `corpus.Corpus(...)` and `experiment.train_ready(...)` functions. Is there an API available for me to apply the preprocessing steps in `corpus.Corpus(...)` to a single wav file and then use something similar to `experiment.transcribe(...)` to get the predicted model transcription?

cc: @oadams

**Comment by oadams** [comment]
Hi, did you face issues with [this approach?](https://persephone.readthedocs.io/en/latest/quickstart.html#saving-and-loading-models-transcribing-untranscribed-data)

**Comment by toonday** [comment]
Yes I did. I was able to add a flag to support a custom behavior of loading a corpus with no train/test/valid data only untranscribed data for inference.

**Comment by shuttle1987** [comment]
@toonday were you able to make progress? It would be good to know what you tried and what issues you encountered.
